File: PartySpa/src/app/_shared/interface/party.ts
export interface Party {
  partyId: number;
  partyName: string;
  partyDate: Date;
  beginTime: string;
  endTime: string;
  marry: number;
  persons: number;

  manAmt: number;
  manEducaton: number;
  manAge1: number;
  manAge2: number;
  womanAmt: number;
  womanEducaton: number;
  womanAge1: number;
  womanAge2: number;
  earlyDate?: Date;
  earlyManAmt?: number;
  earlyWomanAmt?: number;
  womenAmt?: number;

  restaurant: string;
  addressNo: string;
  busNote: string;
  notes: string;
  pictureUrl: string;
}

File: PartyApi/Models/Party.cs
using System;
using System.Collections.Generic;

namespace PartyApi.Models
{
    public partial class Party
    {
        public Party()
        {
            Activity = new HashSet<Activity>();
            Liker = new HashSet<Liker>();
            Message = new HashSet<Message>();
            PartyPhoto = new HashSet<PartyPhoto>();
        }

        public int PartyId { get; set; }
        public string PartyName { get; set; }
        public DateTime PartyDate { get; set; }
        public string BeginTime { get; set; }
        public string EndTime { get; set; }
        public int Marry { get; set; }
        public int Persons { get; set; }
        public int ManAmt { get; set; }
        public int ManEducaton { get; set; }
        public int ManAge1 { get; set; }
        public int ManAge2 { get; set; }
        public int WomanAmt { get; set; }
        public int WomanEducaton { get; set; }
        public int WomanAge1 { get; set; }
        public int WomanAge2 { get; set; }
        public DateTime EarlyDate { get; set; }
        public int EarlyManAmt { get; set; }
        public int EarlyWomanAmt { get; set; }
        public int? WomenAmt { get; set; }
        public string PictureUrl { get; set; }
        public string Restaurant { get; set; }
        public string AddressNo { get; set; }
        public string BusNote { get; set; }
        public string Notes { get; set; }
        public int? WriteType { get; set; }
        public DateTime? WriteTime { get; set; }
        public string WriteUser { get; set; }
        public string WriteIp { get; set; }

        public virtual ICollection<Activity> Activity { get; set; }
        public virtual ICollection<Liker> Liker { get; set; }
        public virtual ICollection<Message> Message { get; set; }
        public virtual ICollection<PartyPhoto> PartyPhoto { get; set; }
    }
}
